File: backend/DataAccess/DaoGenerated/システム共有/tm_afkikan_subDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             医療機関サブマスタ
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tm_afkikan_subDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tm_afkikan_sub";
		public string kikancd { get; set; }                       //医療機関コード（自治体独自）
		public string jissijigyo { get; set; }                    //医療機関・事業従事者（担当者）事業コード
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tm_afkikan_subDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             医療機関サブマスタ
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace Jbd.Gjs.Db
    Public Class tm_afkikan_subDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tm_afkikan_sub"
        Public Property kikancd As String                       '医療機関コード（自治体独自）
        Public Property jissijigyo As String                    '医療機関・事業従事者（担当者）事業コード
    End Class
End Namespace
